webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js

```diff
@@ -155,6 +167,7 @@ export const ErrataTab = () => {
     results,
     metadata,
     idColumn: 'errata_id',
+    fetchItemsQuery: fetchItemsQuery(),
```

Comment:
```suggestion
    dropdownsQuery: dropdownsQuery(),
```
Since `searchQuery` is what the user entered in the search box, I feel `dropdownsQuery` or `filterDropdownsQuery` may be a better name here, since it's what the user has selected in the dropdowns.

Reply:
I don't like to include component type in the name. Suppose in the future we'd like to add a checkbox to the filters then the name would become not appropriate.
How about `filtersQuery`?

Reply:
I'm fine with `filtersQuery` :+1: